tests/visibility_attr/child/print_gazelle.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# --- begin runfiles.bash initialization v2 ---
```

Comment:
Out of curiosity, why did you implement this in bash? Most (all?) of the testing in the rest of the repo is in go, and it seems to me like it would be a pretty simple test to add that way.

Reply:
I can switch it to Go. My brain was thinking rule change.

Reply:
It just occurred to me that I could just use `build_test` under `child`. Does that work for you?

Reply:
Works great for me. Thanks!